### PR TITLE
Seting up dependabot! 

### PR DESCRIPTION
**Pull request by l3gacyb3ta**
Hopefully this works.

### CODE REVIEW

**Comment by l3gacyb3ta** [comment]
I don't _think_ this should break anything...

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "" # See documentation for possible values
```
Comment (cjdenio) [inline .github/dependabot.yml]: I think this has to be set to `yarn`

**Review by cjdenio — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (l3gacyb3ta) [reply .github/dependabot.yml]: will do!

**Comment by cjdenio** [comment]
Just kidding, it has to set to `npm`. Sorry!

**Comment by l3gacyb3ta** [comment]
Yep, that's it caleb!

**Review by cjdenio — APPROVED** [review]
thanks!

**Pull request merged**